test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPublishTests.cs

```diff
@@ -453,6 +453,67 @@ public void DnuPublishConsoleApp(string flavor, string os, string architecture)
 
         [Theory]
         [MemberData(nameof(RuntimeComponents))]
+        public void DnuPublishConsoleAppWithNoSourceOptionNormalizesVersionNumber(string flavor, string os, string architecture)
+        {
+            var runtimeHomeDir = _fixture.GetRuntimeHomeDir(flavor, os, architecture);
+
+            var projectStructure = @"{
+  '.': ['project.json', 'Config.json', 'Program.cs'],
+  'packages': {}
+}";
+            var expectedOutputStructure = @"{
+  'approot': {
+    'global.json': '',
+    'packages': {
+      'PROJECT_NAME': {
+        '1.0.0': {
+          '.': ['PROJECT_NAME.1.0.0.nupkg', 'PROJECT_NAME.1.0.0.nupkg.sha512', 'PROJECT_NAME.nuspec'],
+          'root': {
+            '.': ['Config.json', 'project.json', 'project.lock.json']
+          },
```

Comment:
nit: can be simplified as
```json
'root': ['Config.json', 'project.json', 'project.lock.json']
```
if you just want a folder containing a list of empty files